### ISSUE TITLE
It fails with accented words

### ISSUE THREAD

**Issue by qgustavor** [leandrodvd/spellchecker_ptbr#1]
```javascript
const spellchecker = require('spellchecker_ptbr')
spellchecker.spellcheck('bênção pão sabão então irmão órgão sótão', (err, typos) => {
  console.log(typos)
})

/* [ { word: 'bênção', suggestions: [], positions: [ [Object] ] },
{ word: 'pão', suggestions: [ 'p�o' ], positions: [ [Object] ] },
{ word: 'sabão',
  suggestions: [ 'sab�o' ],
  positions: [ [Object] ] },
{ word: 'então',
  suggestions: [ 'ent�o' ],
  positions: [ [Object] ] },
{ word: 'irmão',
  suggestions: [ 'irm�o' ],
  positions: [ [Object] ] },
{ word: 'órgão', suggestions: [], positions: [ [Object] ] },
{ word: 'sótão', suggestions: [], positions: [ [Object] ] } ] */
```

Seems that's an encoding issue: [pt_BR.dic](https://github.com/leandrodvd/spellchecker_ptbr/blob/master/dic/pt_BR.dic?raw=true) is encoded with Windows-1252, not UTF-8. Fixing the encoding will solve this issue.

**Comment by leandrodvd** [comment]
Hey sorry for not noticing this issue before. I'll take a look on that. Tks for opening the issue.

**Comment by leandrodvd** [comment]
Fixed in version 0.2.0
